### ISSUE TITLE
Snoozy isn't listed in the Play store

### ISSUE THREAD

**Issue by steebubu** [mudar/SnooZy#20]
https://play.google.com/store/apps/details?id=ca.mudar.snoozy doesn't resolve to an app on the play store. Was the app removed?

**Comment by da2x** [comment]
Link works for me here in Norway.

**Comment by da2x** [comment]
I get a 404 if I’m not logged in to my Google Account, however.

**Comment by steebubu** [comment]
https://github.com/mudar/SnooZy/raw/master/APK/SnooZy.apk turns up a 404 :)

Maybe the APK in the play store is geo restricted? I'm in the US, and definitely logged in.

**Comment by mudar** [comment]
Yes, the app was removed from the playstore. APK is still available here on github: https://github.com/mudar/SnooZy/tree/master/APK

I removed the app because of the smart-lock issues (ex #16, #17 and #18), which became more common with the trend for fingerprint unlock. It seems android P fixes the issue, so I might update the app after that.

Thanks for your comment, it feels good (sort-of) to see that some users miss snoozy :)
